src/rgw/rgw_admin.cc

```diff
@@ -4817,7 +4817,11 @@ int main(int argc, char **argv)
     string user_str = user_id.to_str();
     int ret = store->cls_user_get_header(user_str, &header);
     if (ret < 0) {
-      cerr << "ERROR: can't read user header: " << cpp_strerror(-ret) << std::endl;
+      if (ret == -2) { /* in case of ENOENT */
+        cerr << "Error: There is no bucket/object in the cluster. Please create buckets/object first." << std::endl;
```

Comment:
It's not an error, so no need to use 'Error:'. Also, is this always correct? What if another user have data, aren't we going to get the same error. And there's no need for the second part of the sentence (no need to instruct users what to do).